server/lib/topic_cache.py

```diff
@@ -154,7 +191,32 @@ def load_files(fpath_list: List[str], name_overrides: Dict,
 
 
 def load(name_overrides: Dict) -> Dict[str, TopicCache]:
-  topic_cache_map = {}
+  topic_cache_map: dict[str, TopicCache] = {}
```

Comment:
nit: localfile consistency `Dict`?

Reply:
Using the built-in types in this file instead of the [deprecated](https://docs.python.org/3/library/typing.html#typing.Dict) `typing.*` types.

Reply:
We need to do a bulk s/Dict/dict/ then for consistency